validator/src/bin/solana-test-validator.rs

```diff
@@ -312,12 +312,13 @@ fn main() {
                 ),
         )
         .arg(
-            Arg::with_name("accountsdb_plugin_config")
-                .long("accountsdb-plugin-config")
+            Arg::with_name("geyser_plugin_config")
+                .long("geyser-plugin-config")
+                .alias("accountsdb-plugin-config")
```

Comment:
backward compatibility?

Reply:
Yep, I figured we might as well avoid breaking the validator interface, even though I guess users will have to redeploy on that update anyway.